Nim/Nim.sln

```diff
@@ -0,0 +1,16 @@
+﻿
```

Comment:
You have created way too many solutions. You were supposed to have one solution with multiple projects. Please re-watch the Loom on Projects & Solutions or ask me for help.